src/App.js

```diff
@@ -1,28 +1,23 @@
-import logo from './logo.svg';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Great job working on this milestone. I don't think you should disable eslint, rather you should fix the errors that arise as the linters are there to make your robust and professional.